src/man/firejail.1.in

```diff
@@ -1887,11 +1887,24 @@ Disable printers.
 
 .TP
 \fB\-\-noprofile
-Do not use a security profile.
+Do not load any security profile at all.
 .br
 
 .br
-Example:
+Note that certain restrictions are specified in the source code rather than in
+profiles.
```

Comment:
Maybe clarify that some restrictions are enabled by default and can be lifted with `keep-*`, `writable-*`, `noblacklist`, `allow-*`, ... and some are enabled unconditionally.

Reply:
> Maybe clarify that some restrictions are enabled by default and can be lifted
> with `keep-*`, `writable-*`, `noblacklist`, `allow-*`, ...

For the sake of maintenance, I think it would be better in general to avoid
mirroring all the commands from noprofile.profile (other than in
profile.template).

Likewise, if there is to be a list of all other "allow" commands, I think it
would make more sense to put that in its own section rather than inside
the description of `--noprofile`.

But I added a note about the more generic ones ones: `--noblacklist`,
`--nowhitelist` and `--ignore`.

> and some are enabled unconditionally.

Added a part about this.